app/views/posts/_post.json.jbuilder

```diff
@@ -0,0 +1,19 @@
+json.post do
+	json.text        	post.long
+	json.random_id		"47"
+	json.num_comments 	post.comments.count
+	json.id     		post.id
+end
+
+json.user do
+	json.first     	post.project.user.firstName
```

Comment:
should be `firstName` and `lastName`, first and last are actual functions in ruby and in jQuery that grab the first or last element from an array, so they are misleading in this context.